src/Compilers/CSharp/Portable/Syntax/Syntax.xml

```diff
@@ -3357,6 +3357,10 @@
     <Field Name="Keyword" Type="SyntaxToken" Override="true">
       <ContextualKind Name="RecordKeyword"/>
     </Field>
+    <Field Name="RecordModifier" Type="SyntaxToken" Optional="true">
+      <Kind Name="ClassKeyword"/>
+      <Kind Name="StructKeyword"/>
+    </Field>
```

Comment:
nit... (and you're probably right to do it thsi way).  but elsewhre we would say ClassOrStructKeyword.  

*however*... i could see us doing more stuff here, so perhaps RecordModifier is appropriate.  Consider TypeModifier as well?